build.gradle

```diff
@@ -0,0 +1,95 @@
+plugins {
+    id "com.github.johnrengelman.shadow" version "5.2.0"
+    id "de.undercouch.download" version "4.0.4"
+    id "net.researchgate.release" version "2.8.0"
+}
+
+ext.ballerinaLangVersion = project.ballerinaLangVersion
+
+allprojects {
+    group = project.group
+    version = project.version
+
+    apply plugin: 'maven-publish'
+
+    repositories {
+        mavenLocal()
+
+        maven {
+            url = 'https://maven.wso2.org/nexus/content/groups/wso2-public/'
+        }
+
+        maven {
+            url = 'https://repo.maven.apache.org/maven2'
+        }
+
+        maven {
+            url = 'https://maven.pkg.github.com/ballerina-platform/ballerina-lang'
+            credentials {
+                username "kapilraajP"
+                password "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"
```

Comment:
Lets remove these credentials and use tokens from env which needs to be set from GH secrets